app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
+    post = user.posts[params[:id].to_i - 1]
+    like = Like.new
+    like.author = current_user
+    like.post = post
```

Comment:
- [ ] in this method you can also use the current_user method created in the application controller.